cmake/GIT.cmake

```diff
@@ -62,4 +77,4 @@ if (DEFINED ENV{GIT_RELEASE})
     set(GIT_RELEASE "$ENV{GIT_RELEASE}")
 endif ()
 
-message(STATUS "Injected git values: ${GIT_COMMIT} (${GIT_RELEASE}) ${GIT_HASH}")
+message(STATUS "Injected git values: ${GIT_COMMIT} (${GIT_RELEASE}) ${GIT_HASH} ${GIT_MODIFIED}")
```

Comment:
This `${GIT_MODIFIED}` returns `0` or `1`, i see this being confusing, maybe adding a string saying `modified: 0/1` or removing this value completely would help.

Reply:
You mean just changing the output that's printed, right? Not changing the value of the variable itself. I figured that because there's a space between these it would be ok if you know what you're reading, but I suppose it's better to be more descriptive.
As per another option which'd be removal... not sure, I think it might be more profitable to keep this in case of any potential debugging in the future, so I guess I'll just add a little note like you proposed.